examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,507 @@
+  case chatMessageEdited(AzureCommunicationChat.ChatMessageEditedEvent)
+  case chatMessageDeleted(AzureCommunicationChat.ChatMessageDeletedEvent)
+  case chatThreadCreated(AzureCommunicationChat.ChatThreadCreatedEvent)
+  case chatThreadPropertiesUpdated(AzureCommunicationChat.ChatThreadPropertiesUpdatedEvent)
+  case chatThreadDeleted(AzureCommunicationChat.ChatThreadDeletedEvent)
+  case participantsAdded(AzureCommunicationChat.ParticipantsAddedEvent)
+  case participantsRemoved(AzureCommunicationChat.ParticipantsRemovedEvent)
+}
+public struct UpdateChatMessageRequest : Swift.Codable {
+  public let content: Swift.String?
+  public init(content: Swift.String? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct ChatMessageContent : Swift.Codable {
+  public let message: Swift.String?
+  public let topic: Swift.String?
+  public let participants: [AzureCommunicationChat.ChatParticipant]?
+  public let initiator: AzureCommunicationCommon.CommunicationIdentifier?
+  public init(message: Swift.String? = nil, topic: Swift.String? = nil, participants: [AzureCommunicationChat.ChatParticipant]? = nil, initiator: AzureCommunicationCommon.CommunicationIdentifier? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct MicrosoftTeamsUserIdentifierModel : Swift.Codable {
+  public let userId: Swift.String
+  public let isAnonymous: Swift.Bool?
+  public let cloud: AzureCommunicationChat.CommunicationCloudEnvironmentModel?
+  public init(userId: Swift.String, isAnonymous: Swift.Bool? = nil, cloud: AzureCommunicationChat.CommunicationCloudEnvironmentModel? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+final public class ChatError : Swift.Codable, Swift.Error {
+  final public let code: Swift.String
+  final public let message: Swift.String
+  final public let target: Swift.String?
+  final public let details: [AzureCommunicationChat.ChatError]?
+  final public let innerError: AzureCommunicationChat.ChatError?
+  public init(code: Swift.String, message: Swift.String, target: Swift.String? = nil, details: [AzureCommunicationChat.ChatError]? = nil, innerError: AzureCommunicationChat.ChatError? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  final public func encode(to encoder: Swift.Encoder) throws
+  @objc deinit
+}
+public struct SendChatMessageRequest : Swift.Codable {
+  public let content: Swift.String
+  public let senderDisplayName: Swift.String?
+  public let type: AzureCommunicationChat.ChatMessageType?
+  public init(content: Swift.String, senderDisplayName: Swift.String? = nil, type: AzureCommunicationChat.ChatMessageType? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
```

Comment:
We've probably discussed this before, but why does create use a `SendRequest` object but update just does that inline and doesn't expose the `UpdateRequest` object?

Reply:
With update you can only update the string content so we do it inline instead of constructing the request. This will change in preview6 though because you can also edit metadata, in preview6 we do use UpdateChatMessageRequest

Reply:
You should update this to use UpdateChatMessageRequest then. I have a hard time believing the ArchBoard would approve breaking change immediately after GA and this is entirely foreseeable.

Reply:
it would not be a breaking change if this becomes an overload. The other SDKs that were GAd before went though the review already. With this said though, given that this has not been released, I agree that it is better to do it now.

Reply:
We received previous feedback to change arguments to be inline so that was the original motivation. This was before preview6 came out, I can update it now

Reply:
You could do an overload, true, but it will may add customer confusion. Let me know how you'd like to proceed.